Tests/SwiftDocCTests/Indexing/NavigatorIndexTests.swift

```diff
@@ -243,6 +243,28 @@ Root
         XCTAssertTrue(validateTree(node: treeWithAttributesNonAtomic.root, validator: attributesValidator), "The tree lacks the correct attributes.")
     }
     
+    func testLoadingNavigatorIndexDoesNotCacheReferences() throws {
```

Comment:
This actually passes even without the change in this PR (I think we recently removed any usage of `ResolvedTopicReference` in the navigator index in #166) but I think this is still a useful test to ensure we don't regress.